README.md

```diff
@@ -1 +1,2 @@
-# yosemite
\ No newline at end of file
+# yosemite
+# this comment added on feature1 branch initially
```

Comment:
hi this is from grandcyngal will the comment tell you that.  i think your comment is great, misthiker!